docker/firewheel.dockerfile

```
@@ -73,12 +74,15 @@ RUN bash -c "python3.10 -m venv /fwpy \
 
 # Configure Firewheel
 RUN bash -c "source /fwpy/bin/activate  && \
-    mkdir -p /var/log/minimega && \
+    mkdir -p ${LOGGING_ROOT_DIR} && \
+    mkdir -p ${MM_FILEPATH} && \
+    mkdir -p ${OUTPUT_DIR}/vm_resource_logs && \
+    mkdir -p ${OUTPUT_DIR}/transfers && \
     mkdir -p ${OUTPUT_DIR} && \
```

Minor suggestion to move creating `OUTPUT_DIR` above the others.

```suggestion
    mkdir -p ${OUTPUT_DIR} && \
    mkdir -p ${OUTPUT_DIR}/vm_resource_logs && \
    mkdir -p ${OUTPUT_DIR}/transfers && \
```
Arguably it could just be removed, because the directory must exist already for `vm_resource_logs` and `transfers` to be created there, and the `-p` option in those commands should create it if it doesn't exist. But I think it makes sense to leave it as an obvious step by itself (i.e., always make sure that the output directory is created).

---

Wait, sorry, did you have a reason for leaving the `mkdir -p ${OUTPUT_DIR}` step after the other two though? It seems like if that's a step by itself it should be first unless I'm missing something